src/formatDate.js

```diff
@@ -50,7 +50,40 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = toFormat[3];
+  const dataArray = date.split(fromFormat[3]);
+  let toDate = '';
+
+  if (fromFormat.includes('YYYY') && toFormat.includes('YY')) {
+    const index = fromFormat.indexOf('YYYY');
+
+    fromFormat[index] = 'YY';
+    dataArray[index] = dataArray[index].slice(2);
+  }
+
+  if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
```

Comment:
```suggestion
  } else {
```
No need to write inverted conditions - just use else.

Reply:
corrected